src/main/scala/dcos/metronome/MetronomeConfig.scala

```diff
@@ -93,7 +95,7 @@ class MetronomeConfig(configuration: Configuration) extends JobsConfig with ApiC
       "--kill_retry_timeout" -> Some(killRetryTimeout.toString))
       .collect { case (name, Some(value)) => (name, value) }
       .flatMap { case (name, value) => Seq(name, value) }
-    new AllConf(options.toSeq ++ flags.flatten)
+    new AllConf(options.toList ++ flags.flatten)
```

Comment:
why has this changed to `toList`?

Reply:
this is an issue of double packaging... I think :) .   The Seq out of toSeq was scala when we needed mesosphere.marathon.Seq.   The issue is the `object marathon` for `package mesosphere` lives in the marathon project...   The toList was a way to circumvent the conversation mismatch.

Reply:
it should have been to[Seq] -> changed